stock_account_operating_unit/tests/test_stock_account_operating_unit.py

```diff
@@ -346,15 +340,15 @@ def test_pickings(self):
             operating_unit=None,
             expected_balance=expected_balance,
         )
-        # GL account ‘Inventory’ has balance 0 on OU main_operating_unit
-        expected_balance = 0.0
+        # GL account ‘Inventory’ has balance 1 on OU main_operating_unit
+        expected_balance = 1.0
         self._check_account_balance(
             self.account_inventory.id,
             operating_unit=self.ou1,
             expected_balance=expected_balance,
         )
         # GL account ‘Inventory’ has balance 2 on OU b2c
```

Comment:
In my opinion the comment is correct, it should be 1 (the previous reception) + 1 (this internal transfer).

So the removal of the action_done method probably made this to have balance 1 in b2c instead of 2. Perhaps we should not remove the method but check we are not moving from in/out the company to avoid the double posting. So the override of the action_done method would be only for this inter-OU transfers between internal locations. What do you think?

Reply:
I'd say you are most probably right. I guess the best approach is readding the action_done and then checking that we are only applying it to the inter-ou internal transfers as you mentioned